README.md

```diff
@@ -1,11 +1,34 @@
 # react-scroll-sync
-Synced scroll position across multiple scrollable elements
 
-## Demo
+Synced scroll position across multiple scrollable elements.
 
-http://react-sync-scroll.netlify.com/
+## Installation
+
+```shell
+$ npm install react-scroll-sync
+```
+
+## Usage
```

Comment:
Why adding example here since we already have much more examples and they are even interactive?

Reply:
Sometimes you don't want to have/go to the whole documentation and need to see directly the API, mostly that this doc is forwarded on npm. So having the direct code helps you to understand how easy or not it is to use it.

Reply:
It doesn’t help with maintenance, though. Also it is easy to publish a broken example.

Reply:
Okay, I can remove that if you prefer. :)

Reply:
Ideally I think we could just use one readme.md for GitHub and styleguide. Do you think you could work on that?

Reply:
I love online doc but your package seems quite simple not to have a dedicated documentation. I think a README is enough yes. :)